src/client/client4.js

```diff
@@ -1095,14 +1302,15 @@ export default class Client4 {
     };
 
     logClientError = async (message, level = 'ERROR') => {
-        const body = {
-            message,
-            level
-        };
+        if (!this.enableLogging) {
```

Comment:
hmmm, interesting we are using this in RN to log any error being caught, should we force this in the RN app?